samples/WOCCatalog/WOCCatalog/ImagesViewController.m

```diff
@@ -38,6 +40,180 @@ + (UIImage*)scaleImage:(CGImageRef)imageRef scaledRect:(CGRect)rect quality:(CGI
     return scaledImage;
 }
 
++ (void)initCGImageFormat:(CGImageRef)imageRef formatInfo:(vImage_CGImageFormat*)formatInfo {
```

Comment:
For the purposes of this feature, it would make sense to add this to a new page instead.  My plan is to move all the non UX related work to their own samples, and having it in a separate page would make it easier.  Also will help keep the image view page relatively simple.